.gitignore

```diff
@@ -15,3 +15,4 @@ bin/*
 **/gofiles/*
 **/*.zip
 **/DSP*
+.idea/
```

Comment:
Please remove this line. See header at top of file.

Reply:
done